.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "inference/thirdparty/shark-runtime"]
 	path = inference/thirdparty/shark-runtime
-	url = https://github.com/NodLabs/shark-runtime.git
+	url =https://github.com/nod-ai/SHARK-Runtime.git
+	branch = shark-06032022
```

Comment:
i think you don't need the branch here since you specify the SHA later. But we can clean up in the next patch.

Reply:
don't you need the branch because thats where I pushed the api updates to?  without it, won't it pull main, which doesn't have the function